executor/include/execution/property/blocking.hpp

```diff
@@ -48,7 +48,7 @@ class blocking_t {
     }
   };
 
-  static constexpr possibly_t possibly{};
+  possibly_t possibly{};
```

Comment:
Why remove static and const too?

Please add a note regarding these variables being constexpr-able

Reply:
Added back const, static was removed as for they `query` methods the properties are pass by reference.